pkgs/by-name/op/openinfraquote-bin/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  autoPatchelfHook,
+  lib,
+  stdenv,
+  fetchurl,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "openinfraquote-bin";
+  version = "v1.10.0";
+
+  src =
+    let
+      selectSystem =
+        attrs:
+        attrs.${stdenv.hostPlatform.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system}");
+      suffix = selectSystem {
+        aarch64-linux = "linux-arm64";
+        x86_64-darwin = "darwin-amd64";
+        x86_64-linux = "linux-amd64";
+      };
+      sha256 = selectSystem {
+        aarch64-linux = "sha256-u1SnjhyPP0Wj6P12Ns19K2yVIN+9Il5eYeXPOtXYHFc=";
+        x86_64-darwin = "sha256-O8oZOWLj83WcjUAuNS8WoUgYEkOMdB7uocZD7vMVIhw=";
+        x86_64-linux = "sha256-27Ceina/cB4+WMwQbawmSWq5i3CL267IpwQkN0onJw0";
+      };
+    in
+    fetchurl {
```

Comment:
Why not build from source?

Reply:
1. They already have pre-build binaries, so it's easier to do it.
2. I plan to build this from source later, but right now that also requires packaging [hll](https://opam.ocaml.org/packages/hll/) and [pds](https://opam.ocaml.org/packages/pds/). I lack the time to package custom OCaml build systems. Might be easier than I think it is, but I'm not familiar with OCaml tooling on Nix.